### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by jtsaito**
I'm adding integration tests instead of proper unit tests because GraphQL requires a close coupling of logic and API.

### CODE REVIEW

File: jest.config.json
Comment (jansiwy) [inline jest.config.json:1]: 2 spaces

File: __tests__/projects.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
+import nock from 'nock'; // https://github.com/nock/nock
+
+import { Octokit } from '@octokit/core';
+import { paginateGraphql } from '@octokit/plugin-paginate-graphql';
+import { RepositoryProjectsManager } from '../projects.js'; // eslint-disable-line import/extensions
+
+const GraphQlOctokit = Octokit.plugin(paginateGraphql);
+const octokit = new GraphQlOctokit({ auth: 'fake-token-value' }); // dont' use default GITHUB_TOKEN token from env
```
Comment (jansiwy) [inline __tests__/projects.js:8]:
```suggestion
const octokit = new GraphQlOctokit({ auth: 'fake-token-value' }); // don't use default GITHUB_TOKEN token from env
```

File: __tests__/projects.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+const rpm = new RepositoryProjectsManager({
+  owner: 'babbel-sandbox',
+  repository: 'test-repo-jsaito-3',
+  octokit,
+});
+
+describe('RepositoryProjectsManager.sync() posts requests to the API', () => {
+  beforeEach(() => {
+    if (!nock.isActive()) {
```
Comment (jansiwy) [inline __tests__/projects.js:18]: why do we need check some condition first?

File: __tests__/projects.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+import { Octokit } from '@octokit/core';
+import { paginateGraphql } from '@octokit/plugin-paginate-graphql';
+import { RepositoryProjectsManager } from '../projects.js'; // eslint-disable-line import/extensions
+
+const GraphQlOctokit = Octokit.plugin(paginateGraphql);
+const octokit = new GraphQlOctokit({ auth: 'fake-token-value' }); // dont' use default GITHUB_TOKEN token from env
+
+const rpm = new RepositoryProjectsManager({
+  owner: 'babbel-sandbox',
```
Comment (jansiwy) [inline __tests__/projects.js:11]: please replace all specific identifiers, e.g. orgs/companies might be always `acme`

File: __tests__/projects.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  repository: 'test-repo-jsaito-3',
```
Comment (jansiwy) [inline __tests__/projects.js:12]:
```suggestion
  repository: 'example',
```

(also in other places)

File: __tests__/projects.js [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+                nodes: [
+                  {
+                    id: 'PVT_kwDOAnsQgs4AP9Qq',
+                    title: 'layer-200/foo',
+                  },
+                  {
+                    id: 'PVT_kwDOAnsQgs4AP9Qo',
+                    title: 'layer-100/grogu',
+                  },
+                ],
```
Comment (jansiwy) [inline __tests__/projects.js:63]: I don't understand this pair "foo" and "grogu"... can you find something matching?

File: __tests__/projects.js (hunk 2 above)
Comment (jtsaito) [reply __tests__/projects.js:18]: @jansiwy i thought it was needed but apparently it's not.

**Review by jansiwy — APPROVED** [review]

**Pull request merged**